recipes/sparse/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sparse" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "bb433d9adf16b0163436e3890bd1d14694d813697cacabd38434830f7901885e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >= 1.14
+    - scipy >= 0.19
+    - numba >= 0.37
+
+test:
```

Comment:
You can run `pytest` here using the `command` section of `test`:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

You'll need to add `pytest` and any other testing-only requirement in the `requires` section:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-requirements

Also being able to run the test suite depends on the test files being included in the source distribution.